### ISSUE TITLE
The ARI result of DLPFC 151676 dataset 

### ISSUE THREAD

**Issue by HappyLife-together** [QIFEIDKN/STAGATE#6]
Following the tutorial you provided, I ran your method on DLPFC151676 dataset, and the ARI obtained from my training was only **0.58**. Deviations from the ARI results（ARI=**0.60**） in your tutorial （https://stagate.readthedocs.io/en/latest/T1_DLPFC.html）. And the visualization results of PAGA also have some biases. I would like to ask what might be causing this. The data parameters are all provided as per your tutorial.  The data source is downloaded from the URL provided in your tutorial. In terms of environment configuration, tensorflow==1.15.0.

**Comment by xiao-kong-long** [comment]
hello, I wanna know the version of your python and rpy2, I meet some struggles in setting env

**Comment by dknqifei** [comment]
Limited by the framework of tensorflow1, STAGATE itself has a certain randomness. We discussed the issue of randomness in our paper, which we do not think has a particularly significant effect on the results.

**Comment by HappyLife-together** [comment]
> Limited by the framework of tensorflow1, STAGATE itself has a certain randomness. We discussed the issue of randomness in our paper, which we do not think has a particularly significant effect on the results.

I tried training multiple times, but got the same result, ARI=0.58. This doesn't seem like the effect of randomness.

**Comment by HappyLife-together** [comment]
> Limited by the framework of tensorflow1, STAGATE itself has a certain randomness. We discussed the issue of randomness in our paper, which we do not think has a particularly significant effect on the results.



> hello, I wanna know the version of your python and rpy2, I meet some struggles in setting env

In my env, python==3.7, rpy2==3.4.5, hope it helps you.

**Comment by xiao-kong-long** [comment]
> 

thanks very much!

**Comment by ly15296191253** [comment]
It's necessary to copy the offical evn completely？I install the 'scanpy==1.9.1' and 'tensorflow==1.15.0' ,and then run the code as tutorial  on the dataset DLPFC151676. Finally,I get the ARI only 0.43. I don't have a idea where the problem is,may you give me some suggestion?

**Comment by yuxiaokang-source** [comment]
I also meet the same problem with STAGATE, I also install the 'scanpy==1.9.1' and 'tensorflow==1.15.0' ,when I set 10 different random seed for STAGATE initization with 151676 dataset , the result of ARI is very unstable, 
<img width="348" alt="image" src="https://user-images.githubusercontent.com/59059267/193806791-5c251e4d-2cb9-400d-b1b3-983387d9cf34.png">

**Comment by BinDuan** [comment]
I have the similar problem. I followed the same code the tutorial gave, but I can't obtain the same ARI, too much lower than the tutorial gave.

**Comment by CNSLyk7** [comment]
I also have the similar issue, the ARI that I calculated is much lower than that in the paper. Any suggestions to solve this would be greatly appreciated!
